test/common/components/DropDown.unit.spec.jsx

```diff
@@ -1,24 +1,49 @@
 import React from 'react';
-import SkinDeep from 'skin-deep';
+import ReactDOM from 'react-dom';
 import { expect } from 'chai';
+import sinon from 'sinon';
 
 import DropDown from '../../../src/js/common/components/DropDown.jsx';
 
 describe('<DropDown>', () => {
+
+  const clickHandler = sinon.stub();
+  const container = window.document.createElement('div');
+
+  container.addEventListener = sinon.spy(container.addEventListener.bind(container));
+  container.removeEventListener = sinon.spy(container.removeEventListener.bind(container));
+
   const props = {
     buttonText: 'Button text',
-    clickHandler: () => {},
+    clickHandler,
+    container,
     contents: (<h1>Hi</h1>),
     cssClass: 'testClass',
     isOpen: true,
     icon: (<svg><rect x="50" y="50" width="50" height="50"/></svg>),
     id: 'testId'
   };
 
-  const tree = SkinDeep.shallowRender(<DropDown {...props}/>);
+  ReactDOM.render(<DropDown {...props}/>, container);
```

Comment:
Out of curiosity, why was `SkinDeep` not working for you?

I'm guessing either because `shallowRender()` doesn't go through the whole React lifecycle or it doesn't render children without `.dive()`ing (or both), but I'd like to confirm since I haven't had to use `ReactDOM.render()` in a unit test yet.

Reply:
Exactly - I really wanted to have a test case written for the `componentWillUnmount` lifecycle method to confirm that the event handler on the document is being removed correctly. It just looks like a potential memory leak to me otherwise.

Reply:
Awesome. Good work!

Reply:
Thank you!